src/main.rs

```diff
@@ -411,4 +416,23 @@ INCLUDE device.x
             vec!["device.x"]
         );
     }
+    #[test]
+    fn parse_flags() {
```

Comment:
could you change the name of this test to `parse_attributes`? the `(rx)` part of the line is referred to as "list of attributes" in the [linker section documentation](https://sourceware.org/binutils/docs/ld/MEMORY.html)